manager/integration/tests/common.py

```diff
@@ -2505,7 +2505,7 @@ def wait_for_backup_completion(client, volume_name, snapshot_name=None,
     return v
 
 
-def wait_pod_auto_attach_after_first_backup_completion(
+def wait_pod_attach_after_first_backup_completion(
```

Comment:
NIT: This renaming should be in the 1st commit

Reply:
Addressed.